metadata_gui_panel.py

```
@@ -319,12 +419,22 @@ def __init__(self, comic_metadata: MetadataInfo):
 
         total_creator_info = ""
         for role, creators_list in comic_metadata.creators:
+            if len(creators_list) == 1 and (
+                creators_list[0] == "MISSING" or creators_list[0] == "<MISSING>"
+            ):
+                continue
+            creators_list = [
```

Ruff (0.15.15)

[warning] `for` loop variable `creators_list` overwritten by assignment target

(PLW2901)